### DISCUSSION TITLE
Guidance on existing path

### DISCUSSION THREAD

**Discussion by d-damien** [organicmaps/organicmaps#4988] [Ideas]
Hello,

OM can provide guidance for a path it has found. It can also display a predetermined path from an existing path file (kml).

But I couldn't find a way to have guidance for a path file. Maybe I missed something ?

I think it would make a lot of sense and complement these features (while not requiring too much work if I'm not mistaken, since it's a path in both cases).

**Use case :** I'm travelling by bike, and despite good pathfinding algorithms, it's still dangerous to go around. So I'm working on a project to collect optimized, tested, safe itineraries between towns. The downside is they're hard to follow as they're usually not the most direct. Hence a need for guidance on existing paths.

LMK if I should open an issue.

Thank you for this awesome software.

**Comment by biodranik** [comment]
Please follow #1360
